openshift/run-2e2-test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+here=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+F8A_API_URL=$(oc get route bayesian-api | awk '{print$2}' | grep -w "${OC_USERNAME}")
+F8A_JOB_API_URL=$(oc get route bayesian-jobs | awk '{print$2}' | grep -w "${OC_USERNAME}")
+F8A_ANITYA_API_URL=$(oc get route bayesian-api | awk '{print$2}' | grep -w "${OC_USERNAME}")
+
+#install python dependencies
+pip install --user -r $here/../integration-tests/requirements.txt
```

Comment:
Do we need this here?

https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/runtest.sh#L9